packages/grid/src/HeaderCell.tsx

```diff
@@ -59,15 +66,25 @@ export const HeaderCellWrapper: React.FC<IHeaderCellWrapper> = ({
     },
     [setChangingColumns, handleMouseMove, handleMouseUp, shouldChangeColumnsWidth],
   );
+
   return (
     <HeaderCell theme={ theme } width={ newWidth } isEmpty={ isEmpty }>
       <HeaderCellContent
         theme={ theme }
-        onMouseDown={ (e) => onMouseDown(e, index) }
         center={ center }
+        onMouseDown={ (e) => onMouseDown(e, index) }
+        onClick={ () => onChangeSort(sortable, sortOrder, index, gridPosition) }
         shouldMovingColumns={ shouldMovingColumns }
       >
         <span>{ isEmpty ? null : text }</span>
+        {
+          sortOrder === 'ask'
```

Comment:
It is better rename sortOrder to sortType or sortDirection, because now it feels like order in which sorting will be applied.
Also 'ask' and 'desk' must be consts or enum.

Reply:
sortOrder is more common. e.g. https://docs.microsoft.com/en-us/dotnet/api/system.data.sqlclient.sortorder?view=netframework-4.8 

I think. sortOrder is ok

Reply:
Ok, then just 'ask' and 'desk' must be consts or enum.